oidc-configure.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# Add a new realm
+cd /opt/keycloak-4.5.0.Final/bin/
+./kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin  --password KEYCLOAKPASS
+./kcadm.sh create realms -s realm=ondemand -s enabled=true -s loginWithEmailAllowed=false -s rememberMe=true
```

Comment:
You can set the ID so that later don't have to parse it out.  Just set the ID to be `ondemand` via `--id=ondemand`

Reply:
Correction, `-s id=ondemand`.